src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Vector64.cs

```diff
@@ -2373,9 +2309,8 @@ public static Vector64<T> Subtract<T>(Vector64<T> left, Vector64<T> right)
         [Intrinsic]
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public static T Sum<T>(Vector64<T> vector)
-            where T : struct
         {
-            T sum = default;
+            T sum = default!;
```

Comment:
This makes me realize this change could actually introduce new warnings in 3rd party code.  That's fine, just something to keep in mind.  I'd been thinking of the removal of the constraint as being completely non-breaking.

Reply:
I don't think that's the case....

This warning is because we made an explicit change to the constraint such that its now unconstrained and now could be `string`, therefore `default` would be invalid for the specified `T`.

Existing callers must be `where T : struct`, so the compiler would consider `T` to be always non-nullable and the result to likewise be always non-nullable.

Reply:
> Existing callers must be where T : struct

Ah, of course.